openchrom/plugins/net.openchrom.nmr.processing.supplier.base/src/net/openchrom/nmr/processing/supplier/base/core/UtilityFunctions.java

```diff
@@ -81,7 +81,19 @@ public int findIndexOfValue(double[] array, double value) {
 				break;
 			}
 		}
-		return index;
+		//
+		int reverseIndex;
+		for(reverseIndex = array.length - 1; reverseIndex > 0; reverseIndex--) {
+			if(Math.abs(array[reverseIndex] - value) < 0.001) {
+				break;
+			}
+		}
+		//
+		if(0 > value) {
```

Comment:
switch constant and value
```suggestion
		if(value< 1) {
```
Switch variable an constant expression